### ISSUE TITLE
Remove application infrastructure code once it is in glue-jupyter

### ISSUE THREAD

**Issue by astrofrog** [spacetelescope/jdaviz#115]
Once https://github.com/glue-viz/glue-jupyter/issues/185 is done, we should remove the application infrastructure code from this repository and make sure everything works fine if we use the infrastructure in glue-jupyter.

**Comment by pllim** [comment]
Is this still relevant?

**Comment by astrofrog** [comment]
Yes, we still need to move the main application code over to glupyter
